data/accounts/es_AR/acctchrt_business.gnucash-xea

```diff
@@ -0,0 +1,844 @@
+  <act:commodity-scu>100</act:commodity-scu>
+  <act:description>Ventas</act:description>
+  <act:parent type="new">59d316be7bc5420cb5c1e51e4c3d41be</act:parent>
+</gnc:account>
+<gnc:account version="2.0.0">
+  <act:name>Otros Ingresos</act:name>
+  <act:id type="new">c9786f2462144b9aba1d598f5867e509</act:id>
+  <act:type>INCOME</act:type>
+  <act:commodity>
+    <cmdty:space>ISO4217</cmdty:space>
+    <cmdty:id>ARS</cmdty:id>
+  </act:commodity>
+  <act:commodity-scu>100</act:commodity-scu>
+  <act:description>Otros Ingresos</act:description>
+  <act:parent type="new">59d316be7bc5420cb5c1e51e4c3d41be</act:parent>
+</gnc:account>
+<gnc:account version="2.0.0">
+  <act:name>Intereses Ganados</act:name>
+  <act:id type="new">01a6e30b82894ec0b8aa7fbb5d9cd7ce</act:id>
+  <act:type>INCOME</act:type>
+  <act:commodity>
+    <cmdty:space>ISO4217</cmdty:space>
+    <cmdty:id>ARS</cmdty:id>
+  </act:commodity>
+  <act:commodity-scu>100</act:commodity-scu>
+  <act:description>Intereses Ganados</act:description>
+  <act:slots>
+    <slot>
+      <slot:key>notes</slot:key>
+      <slot:value type="string"></slot:value>
+    </slot>
+    <slot>
+      <slot:key>placeholder</slot:key>
+      <slot:value type="string">false</slot:value>
+    </slot>
+  </act:slots>
+  <act:parent type="new">59d316be7bc5420cb5c1e51e4c3d41be</act:parent>
+</gnc:account>
+<gnc:account version="2.0.0">
+  <act:name>Egresos</act:name>
+  <act:id type="new">817b0e2d4e0d445398028aae994824ba</act:id>
+  <act:type>EXPENSE</act:type>
+  <act:commodity>
+    <cmdty:space>ISO4217</cmdty:space>
+    <cmdty:id>ARS</cmdty:id>
+  </act:commodity>
+  <act:commodity-scu>100</act:commodity-scu>
+  <act:description>Egresos</act:description>
+  <act:slots>
+    <slot>
+      <slot:key>placeholder</slot:key>
+      <slot:value type="string">true</slot:value>
+    </slot>
+  </act:slots>
+  <act:parent type="new">4daebd948ebd4d74bcc7876818b15fd0</act:parent>
+</gnc:account>
+<gnc:account version="2.0.0">
+  <act:name>Costo de Mercaderías Vendidas</act:name>
+  <act:id type="new">6abc980a25b54aeb8e6bf1dc7304cdf7</act:id>
+  <act:type>EXPENSE</act:type>
+  <act:commodity>
+    <cmdty:space>ISO4217</cmdty:space>
+    <cmdty:id>ARS</cmdty:id>
+  </act:commodity>
+  <act:commodity-scu>100</act:commodity-scu>
+  <act:description>Costo de Mercaderías Vendidas</act:description>
+  <act:parent type="new">817b0e2d4e0d445398028aae994824ba</act:parent>
+</gnc:account>
+<gnc:account version="2.0.0">
+  <act:name>Gastos de Administración</act:name>
+  <act:id type="new">3614a7bd4f3f4d6491d044325ac81108</act:id>
+  <act:type>EXPENSE</act:type>
+  <act:commodity>
+    <cmdty:space>ISO4217</cmdty:space>
+    <cmdty:id>ARS</cmdty:id>
+  </act:commodity>
+  <act:commodity-scu>100</act:commodity-scu>
+  <act:description>Gastos de Administración</act:description>
+  <act:slots>
+    <slot>
+      <slot:key>placeholder</slot:key>
+      <slot:value type="string">true</slot:value>
+    </slot>
+  </act:slots>
```

Comment:
There are no sub accounts. Was this intended?

Reply:
Fixed!

Reply:
Note: I have updated https://wiki.gnucash.org/wiki/Account_Hierarchy_Template#Testing_in_Gnucash